File: app/src/main/java/org/neidhardt/dynamicsoundboard/soundcontrol/events/OnOpenSoundDialogEventListener.java
package org.neidhardt.dynamicsoundboard.soundcontrol.events;

/**
 * Created by eric.neidhardt on 29.05.2015.
 */
public interface OnOpenSoundDialogEventListener
{
	/**
	 * This is called by greenRobot EventBus in case the rename dialog for a certain sound is requested.
	 * @param event delivered OpenSoundRenameEvent
	 */
	void onEvent(OpenSoundRenameEvent event);

	/**
	 * This is called by greenRobot EventBus in case the settings for a certain sound are requested.
	 * @param event delivered OpenSoundSettingsEvent
	 */
	void onEvent(OpenSoundSettingsEvent event);
}

File: app/src/main/kotlin/org/neidhardt/dynamicsoundboard/soundcontrol/events/OnOpenSoundDialogEventListener.kt
package org.neidhardt.dynamicsoundboard.soundcontrol.events

/**
 * File created by eric.neidhardt on 29.05.2015.
 */
interface OnOpenSoundDialogEventListener
{
	/**
	 * This is called by greenRobot EventBus in case the rename dialog for a certain sound is requested.
	 * @param event delivered OpenSoundRenameEvent
	 */
	fun onEvent(event: OpenSoundRenameEvent)

	/**
	 * This is called by greenRobot EventBus in case the settingsButton for a certain sound are requested.
	 * @param event delivered OpenSoundSettingsEvent
	 */
	fun onEvent(event: OpenSoundSettingsEvent)
}
